From da944d96314f16d5207ef1c2b99a59f513403d04 Mon Sep 17 00:00:00 2001
From: Jason Chen <b02280@freescale.com>
Date: Mon, 22 Aug 2011 13:56:41 +0800
Subject: [PATCH 0374/2508] ENGR00155151 imx6q clock: fix ldb and ipu-di clock
 enable register

ipu2-di should use CCGR3 4&5, ldb_di should use 6&7.

Signed-off-by: Jason Chen <b02280@freescale.com>
---
 arch/arm/mach-mx6/clock.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/mach-mx6/clock.c b/arch/arm/mach-mx6/clock.c
index 50a13a4..c037f55 100644
--- a/arch/arm/mach-mx6/clock.c
+++ b/arch/arm/mach-mx6/clock.c
@@ -2630,7 +2630,7 @@ static struct clk ipu2_di_clk[] = {
 	.id = 0,
 	.parent = &pll5_video_main_clk,
 	.enable_reg = MXC_CCM_CCGR3,
-	.enable_shift = MXC_CCM_CCGRx_CG1_OFFSET,
+	.enable_shift = MXC_CCM_CCGRx_CG4_OFFSET,
 	.enable = _clk_enable,
 	.disable = _clk_disable,
 	.set_parent = _clk_ipu2_di0_set_parent,
@@ -2643,7 +2643,7 @@ static struct clk ipu2_di_clk[] = {
 	.id = 1,
 	.parent = &pll5_video_main_clk,
 	.enable_reg = MXC_CCM_CCGR3,
-	.enable_shift = MXC_CCM_CCGRx_CG2_OFFSET,
+	.enable_shift = MXC_CCM_CCGRx_CG5_OFFSET,
 	.enable = _clk_enable,
 	.disable = _clk_disable,
 	.set_parent = _clk_ipu2_di1_set_parent,
-- 
1.7.7.4

